osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -200,5 +201,8 @@ public enum GlobalAction
 
         [Description("Pause / resume replay")]
         TogglePauseReplay,
+
+        [Description("Toggle in-game interface")]
```

Comment:
`In-game interface` is another word to the `HUD` terminology, the other being `score overlay` from the notification. I think we should just use `HUD` for all user facing strings, like `Hold for HUD`. Also `in-game` is redundant, since it's already in the in game keybinding section.

Reply:
There's bound to be more places with inconsistencies like this, so I think this should be a separate pull.